eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -164,7 +164,8 @@ jobs:
 
       - template: ../../../common/pipelines/templates/steps/verify-links.yml
```

Comment:
We should figure out how to condition this step to only run for pull requests.

Reply:
Made the change